src/core/mac/mac.cpp

```diff
@@ -856,10 +856,116 @@ void Mac::HandleBeginTransmit(void)
 
     if (error != kThreadError_None)
     {
+#if OPENTHREAD_ENABLE_PASSING_UP_ACK
+        TransmitDoneTask(mTxFrame, NULL, kThreadError_Abort);
+#else
         TransmitDoneTask(mTxFrame, false, kThreadError_Abort);
+#endif
+    }
+}
+
+#if OPENTHREAD_ENABLE_PASSING_UP_ACK
```

Comment:
Ahh, I see above from your comment that this \#if config is only intended as a temporary workaround.  Ignore my previous comment in that case.